modules/network/basic-cdk/app.py

```diff
@@ -65,6 +65,7 @@ def generate_description() -> str:
     value=stack.to_json_string(
         {
             "VpcId": stack.vpc.vpc_id,
+            "SecurityGroupId": stack._vpc_security_group.security_group_id,
```

Comment:
Is it the default security group ID of the VPC? Curious to know the use-case which you are consuming it for

Reply:
Yes its the security group created [here](https://github.com/awslabs/idf-modules/blob/main/modules/network/basic-cdk/stack.py#L68) and [here](https://github.com/awslabs/idf-modules/blob/main/modules/network/basic-cdk/stack.py#L278).

The security group ID was required for https://github.com/awslabs/aiops-modules/pull/133, so that a VPC config could be specified for the training jobs in sagemaker. We were able to resolve this by creating a new security group in the VPC, but thought it would be worth adding the existing one as an output to this module so it can be consumed for any other use-cases in the future.